openqa-label-known-issues

```diff
@@ -175,8 +171,34 @@ investigate_issue() {
 }
 
 label_issue() {
-    local testurl="${1:?"Need 'testurl'"}"
-    [[ "$dry_run" = "1" ]] && client_prefix="echo"
+    local arg="${1:?"Usage: $0 <job_id> or https://<host>/tests/<job_id>"}"
+
+    scheme=${scheme:-https}
+    local job_id=""
+    local testurl=""
+
```

Comment:
I guess it would be nicer if this was a little bit more compact:
```suggestion
    scheme=${scheme:-https}
    local arg=${1:?"Usage: $0 <job_id> or https://<host>/tests/<job_id>"}
    local job_id= testurl=
```

As you mentioned in your other comment, the error message produced by the `?` syntax is not nice for user-facing errors. Not sure whether it is worth improving, though.

Reply:
if `$1` is not supplied, then arg will never get assigned anything, because the script will be aborted.
And this is without `set -u`:
```
% cat empty.sh
#!/bin/bash
arg=${1:?oops}
echo "arg: $arg"
% ./empty.sh
./empty.sh: line 2: 1: oops
% ./empty.sh 23
arg: 23
```